### PR TITLE
Add FAQ Content

### PR DESCRIPTION
**Pull request by KimSangYeon-DGU**
@zoq 
I added FAQ about building and running the program by reference to [the issues page](https://github.com/zoq/gym_tcp_api/issues) in README.md. 
And I hope it is helpful to users.
If you don't mind, I want it to be merged into master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,3 +138,33 @@ Start the monitor:
...
+<b>2. Failed to fetch record for 'hexpm/poolboy' from registry (using cache)</b>
+
+  - SOL) This was a cache problem, remove the cache file and try again.
+    ```
+    rm "~/.hex/cache.ets"
+    ```
+    
+<b>3. When you run a client, if "what():  Connection refused." error occurs</b>
+  
+  - SOL) If you use your own server instance, changed the line 17 code const std::string host = "kurg.org"; to const std::string host = "127.0.0.1"; in example.cpp and try again.
```
Comment (zoq) [inline README.md]: Do you think ``127.0.0.1`` should be the default?

Comment (KimSangYeon-DGU) [reply README.md]: I think 127.0.0.1 should be the default given that users follow the tutorial in the README.
The reason for this is that the tutorial is supposed to use their own server.

Comment (zoq) [reply README.md]: Agreed, do you like to change the value to "127.0.0.1"? Don't feel obligated, I can do this as well.

**Comment by KimSangYeon-DGU** [comment]
@zoq 
I changed the host's default address into "127.0.0.1" and consequently revised the FAQ.

File: README.md (hunk 1 above)
Comment (KimSangYeon-DGU) [reply README.md]: Okay!!

**Comment by zoq** [comment]
Thanks for the contribution, always nice to improve the documentation and build process.

**Comment by KimSangYeon-DGU** [comment]
Thank you for developing such a good repository.
I'm very happy for contributing to this repository. Thank you. :)

**Pull request merged**